src/components/Markdown.vue

```diff
@@ -24,9 +24,17 @@
 </template>
 
 <script>
-import MarkdownIt from 'markdown-it'
-
-const md = new MarkdownIt()
+import { mystParser } from 'myst-parser'
+import { State, transform, mystToHast, formatHtml } from 'myst-to-html'
+import rehypeStringify from 'rehype-stringify'
+import { unified } from 'unified'
+
+const pipe = unified()
+  .use(mystParser)
+  .use(transform, new State())
+  .use(mystToHast)
+  .use(formatHtml)
+  .use(rehypeStringify)
 
```

Comment:
Can we get away without this `unified` dependency?

Reply:
Will poke

Reply:
Fair enough.

Reply:
I asked on Discord and got asked "Why would you want to?".